db/seeds/02_zipcode_rates.rb

```diff
@@ -0,0 +1,59 @@
+module ZipcodeRateImporter
+  extend self
+  ZIP_MHA_PATH = File.join(Rails.root, 'db', 'seeds', 'zipcode_rate', 'zip_mha.txt')
+  MHA_RATE_PATH = File.join(Rails.root, 'db', 'seeds', 'zipcode_rate', 'mha_rate.txt')
+  ZIP_CITY_PATH = File.join(Rails.root, 'db', 'seeds', 'zipcode_rate', 'zip_city.csv')
+
+  def zip_mha
+    options = {
+      col_sep: '|',
+      headers_in_file: false,
+      user_provided_headers: %w[zip_code mha_code year date],
+      convert_values_to_numeric: false
+    }
+    SmarterCSV.process(ZIP_MHA_PATH, options)
+  end
+
+  def mha_rate
+    options = {
+      col_sep: '|',
+      headers_in_file: false,
+      user_provided_headers: %w[mha_code date mha_rate mha_rate_grandfathered],
+      convert_values_to_numeric: false
+    }
+    SmarterCSV.process(MHA_RATE_PATH, options).group_by { |record| record[:mha_code] }
+  end
+
+  def zip_city
+    options = {
+      col_sep: ',',
+      headers_in_file: true,
+      user_provided_headers: %w[zip_code city state],
+      convert_values_to_numeric: false
+    }
+    SmarterCSV.process(ZIP_CITY_PATH, options).group_by { |record| record[:zip_code] }
+  end
+
+  def insert_data
+    records = []
+    mha_rates = mha_rate
+    zip_cities = zip_city
+    zip_mha.each do |zip_mha_record|
+      mha_rate_data = mha_rates[zip_mha_record[:mha_code]].first.slice(*%i[mha_rate mha_rate_grandfathered])
+      mha_name_data = { mha_name: zip_cities[zip_mha_record[:zip_code]]&.first&.slice(*%i[city state])&.values&.join(', ') }
+      records << ZipcodeRate.new(zip_mha_record.slice(*%i[zip_code mha_code])
```

Comment:
My take on this is the name is just for display under "What is the ZIP Code where you'll be located?" in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9368#issuecomment-382547068.

If that's the case, I'd suggest we drop the name since your observations about "primary" name would likely be confusing. Maybe address in a separate refinement of the tool?

Reply:
@briangryth pulling you in to see if you have any clarifying comments on this one. Essentially, if ^ is correct, what do we do when a zip code covers more than one city/state?

Reply:
according to @saintsoup52 comments there is a way to get the name using crosswalks, but I'm not entirely sure how to do that just yet. I'm also thinking that perhaps the consolidated file that they provide could also include the name so that there is no need to build the dynamically, and it will just be a direct import instead.

Reply:
👍 interested in getting additional details ASAP, but I don't see problems moving forward with this and getting the API schema in place.